dataservice/util/data_gen/data_generator.py

```diff
@@ -0,0 +1,310 @@
+from datetime import datetime
+from dateutil import relativedelta
+import uuid
+import random
+import csv
+import os
+
+from dataservice import create_app
+from dataservice.extensions import db
+from dataservice.api.participant.models import Participant
+from dataservice.api.demographic.models import Demographic
+from dataservice.api.diagnosis.models import Diagnosis
+from dataservice.api.sample.models import Sample
+from dataservice.api.aliquot.models import Aliquot
+from dataservice.api.sequencing_experiment.models import SequencingExperiment
+from dataservice.api.genomic_file.models import GenomicFile
+
+
+class DataGenerator(object):
+    def __init__(self, config_name=os.environ.get('FLASK_CONFIG', 'default')):
```

Comment:
Probably not a good idea to do any logic like this in a function handle. Probably more appropriate to put this in the `__init__.py` where the `DataGenerator` object get's created?

Reply:
You could get rid of config_name as a param and assume it will be read in from the environ, something like: 
```python
class DataGenerator(object):
    def __init__(self):
        self.setup(os.environ.get('FLASK_CONFIG', 'default'))
```
OR leave the config_name as an optional parameter and later on it could be passed in as a cmd line arg to the flask cmd:
 
```python
class DataGenerator(object):
    def __init__(self, config_name=None):
        if not config_name:
            config_name = os.environ.get('FLASK_CONFIG','default')
        self.setup(config_name)
```